### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by millerdev**
Drop obsolete Pythons, add a new one, and ignore failures on 3.7 because the issue is with a sub-dependency of a build requirement and did not appear to be a trivial fix.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -45,10 +45,9 @@ def get_readme():
         'Operating System :: OS Independent',
         'Programming Language :: Python',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
```
Comment (czue) [inline setup.py:50]: scope creep, but wondering if 3.10 support should be a goal too?

**Review by czue — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (millerdev) [reply setup.py:50]: See the commit message https://github.com/dimagi/django-prbac/pull/69/commits/1eb9aa0a233266c4b7a24682b445ef7c9e0754b2

Comment (czue) [reply setup.py:50]: :man_facepalming:
